orttraining/orttraining/python/training/ortmodule/_runtime_inspector.py

```diff
@@ -194,10 +194,25 @@ def _initialize_embedding_padding_inspector(self, model, user_input_names):
             if padding_idx < 0:
                 continue
 
-            if node.input[1] not in self._embedding_graph_input_to_padding_idx_map:
-                self._embedding_graph_input_to_padding_idx_map[node.input[1]] = set()
+            # Check embedding inputs
+            def get_embedding_graph_input(node_arg):
```

Comment:
```suggestion
            def _get_embedding_graph_input(node_arg):
```

Reply:
and move it out of the loop?

Reply:
It's not convenient to move it out, because it need the local variable of `user_input_names` and `self._try_get_node_from_its_output`.